src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+
+export function Navbar() {
+
+    const navbarItems=
+        [{
+            "url": "#SurveyList",
+            "description": "Survey List",
+            "isActive": true
+        }, 
+        {
+            "url": "#CreateSurvey",
+            "description": "Create Survey",
+            "isActive": false
+        }, 
+        {
+            "url": "#MySurveys",
+            "description": "My Surveys",
+            "isActive": false
+        }, 
+        {
+            "url": "#Profile",
+            "description": "Profile",
+            "isActive": false
+        }];
+
+    return (
+        <div>
+            <div className="nav">
+                <a href="#" className="logo" >Survey Genarator</a>
+                {navbarItems.map((item)=>{
+                    if(item.isActive)
+                        return <a className="active" href={item.url}>{item.description}</a>
+                    else
+                        return <a href={item.url}>{item.description}</a>
+                })}
+
+                {/* <a className="active" href="#">Survey List</a>
+                <a href="#">Create Survey</a>
+                <a href="#">My Surveys</a>
+                <a href="#">Profile</a>  */}
+
+            </div>
+            <br />
+        </div>
```

Comment:
You are using two `div` elements. Second `div` element is unnecessary you can use the parent `div`. And you should write a small function for a navigation element. And then just call the function while you iterating the array.  
```suggestion
      <div className="nav">
          <a href="#" className="logo" >Survey Genarator</a>
          {navbarItems.map((item)=>{
             return navigationElement(item);
          })}
    </div>
        
const navigationElement = (item) => {
    if(item.isActive)
        return <a className="active" href={item.url}>{item.description}</a>
    else
        return <a href={item.url}>{item.description}</a>
}
```